src/Path/UploadListings.jsx

```diff
@@ -789,35 +792,34 @@ function UploadListings() {
         }
 
       case "startDate":
-        if (!value && parseInt(listingInput.categoryId) == 3) {
+        if (!value && parseInt(listingInput.categoryId) === 3) {
           return t("pleaseEnterStartDate");
-        } else {
-          return "";
         }
+        return "";
 
       case "endDate":
-        if (parseInt(listingInput.categoryId) === 3) {
-          if (value && new Date(listingInput.startDate) > new Date(value)) {
-            return t("endDateBeforeStartDate");
-          } else {
-            return "";
-          }
-        } else {
-          return "";
+        if (!value && parseInt(listingInput.categoryId) === 3) {
+          return t("pleaseEnterEndDate");
```

Comment:
Why did you remove this validation "new Date(listingInput.startDate) > new Date(value)" ?